modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/index.es.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import React from 'react';
+
+import MultiStepNav from './components/MultiStepNav.es';
+
+const App = ({connected, liferayAnalyticsURL, token}) => {
+	return (
```

Comment:
```suggestion
const App = ({connected, liferayAnalyticsURL, token}) => (
```

It could be good simplify this return because, doesn't have logic :)